### ISSUE TITLE
bundling css with name "**.component.css" not working

### ISSUE THREAD

**Issue by SandipFichadiya** [NativeScript/nativescript-dev-webpack#137]
Hello, there few imports in my app.css like
`@import './components/welcome.component.css';` (I have welcome.component.ts/html file in same directory)

But when I try to bundle my app, I can see that it tries to bundle welcome.component.ts file in my app.css instead of ".css". It gives following error:

```
ERROR in ../~/nativescript-css-loader!./components/welcome.component.ts
    Module build failed: Unknown word (1:10)

    > 1 | import { Component, OnInit } from '@angular/core';
        |          ^

```
If I rename css file to `welcome.css` & import in my app.css, then it works fine.

I seriously don't want to rename my all css. I believe there must be a way. Am I missing something?

**Comment by sis0k0** [comment]
Why are you importing your component-specific stylesheets in `app.css`? You can provide them in your components like that:

```ts
// welcome.component.ts

@Component({
    styleUrls: [ "./welcome.component.css"]
})
class WelcomeComponent {
...
```

That way the css you write in your component-specific css files will be scoped just to the component you use it for and won't pollute the global css (defined in `app.css`).

**Comment by SandipFichadiya** [comment]
@sis0k0  yes I know about it. But I have 2 different themes for app, light & dark. so I have two different css for each components. so I am using [https://github.com/nathanaela/nativescript-themes](url) plugin to apply css at run time.

I guess it won't be good idea to write condition in every component about which theme to apply. So what I am doing is, I will have two css at every component & import them in app.css or app-dark.css. & now when app starts, I apply app.css or app-dark.css based on preference using the above plugin.

This way, I am able to apply themes at app level & keep the css files at component level. I don't know if this approach is good, but now when I try to apply webpack, I am getting the above error.

**Comment by SandipFichadiya** [comment]
@sis0k0 any update on this?

**Comment by Shifs** [comment]
Hi, I have configured webpack . But I am getting the error TypeError: Cannot read property '0' of undefined
# prepare

**Comment by NickIliev** [comment]
@Shifs the error `Cannot read property '0' of undefined` is resolved with version 0.9.0 of nativescript-dev-webpack.

Regarding the usage of the plugin nativescript-themes in NativeScript 3.x.x - see [this logged issue](https://github.com/NathanaelA/nativescript-themes/issues/3) in the plugin's repository
